.mailmap

```diff
@@ -1565,7 +1566,7 @@ ylemkimon <ylemkimon@naver.com> <mail@ylem.kim>
 znxftw <vishnu2101@gmail.com>
 zsc347 <zsc347@gmail.com>
 zzc <1378113190@qq.com>
-zzc <58017008+zzc0430@users.noreply.github.com>
+zzc <58017008+zzc0430@users.noreply.github.com>c
```

Comment:
Why is ‘c’ added in the mailmap?

Reply:
I'm not sure how this happend. I just removed this 'c'.